### ISSUE TITLE
Why regular module import routing module instead of reverse ?

### ISSUE THREAD

**Issue by John0King** [angular/angular#32599]
I'm confusing  why `NomalModule` import the `NormalRoutingModule` instead of   `NormalRoutingModule`  import `NomalModule`  .

From what I think, it'll be more flexiable if the routing module import normal module .

**Comment by alxhub** [comment]
I agree. To me, logically, the module defining the routes should import the module declaring the routed components.
